### PR TITLE
Added enable single mode in config and update treeview binding

### PR DESCRIPTION
**Pull request by sandynmc**
- added config setting for switching between single and double click mode. 
- changed tree view binding in projects tree view only based on the config. 
- updated config UI layout.

### CODE REVIEW

File: project_setup_ui.py [hunk 1]
```diff
@@ -71,11 +71,14 @@ def __init__(self, config, first_time_user):
         self.project_treeview_parents, self.patient_treeview_parents, self.concern_treeview_parents = [], [], []
         project_treeview_height = int(self.window_height * 0.3)
         treeview_width = int(self.window_width * 0.45)
+        # Get single click mode
+        flag = config.get_clickmode()
```
Comment (wsarce) [inline project_setup_ui.py:75]: This variable should be named `clickmode` or something similar.

**Comment by wsarce** [comment]
Because the user interface has been changed and functionality added, the user guide needs to be updated along with the patch number for cometrics.

**Comment by wsarce** [comment]
image
Please ensure the placement of the user interface elements maintains alignment.  The `B` should be aligned with its textbox.

https://github.com/Munroe-Meyer-Institute-VR-Laboratory/cometrics/pull/51/commits/07e1db8a870322d2c756358a31028b45aabcf95c#diff-4336eae8bdda831fd8423bc4a42d1194eaa6205aab1578ce7cfefc58933f3705R355

**Comment by wsarce** [comment]
> * fixed virable name
> 
>     * UI updated
> 
>     * Version no. updated
> 
>     * User guide updated



> * fixed virable name
> 
>     * UI updated
> 
>     * Version no. updated
> 
>     * User guide updated



> * fixed virable name
> 
>     * UI updated
> 
>     * Version no. updated
> 
>     * User guide updated

Where were these changes made?  I only see the original commit on this branch.

**Comment by sandynmc** [comment]
> > * fixed virable name
> >   
> >   * UI updated
> >   * Version no. updated
> >   * User guide updated
> 
> Where were these changes made? I only see the original commit on this branch.

Just pushed the changes.

**Review by wsarce — APPROVED** [review]
- Functionality tested for bug fix
- User guide is correct, updated packaging documents
- Patch number updated

**Pull request merged**